src/pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -46,16 +46,9 @@
 
 
 class CheckpointConnector:
-    def __init__(self, trainer: "pl.Trainer", resume_from_checkpoint: Optional[_PATH] = None) -> None:
+    def __init__(self, trainer: "pl.Trainer") -> None:
         self.trainer = trainer
         self.resume_checkpoint_path: Optional[_PATH] = None
-        # TODO: remove resume_from_checkpoint_fit_path in v2.0
-        self.resume_from_checkpoint_fit_path: Optional[_PATH] = resume_from_checkpoint
```

Comment:
We need to keep this logic for https://pytorch-lightning.slack.com/archives/G016HLWA3C4/p1668215644001489?thread_ts=1668162264.697159&cid=G016HLWA3C4. Renaming it.

It can be done in a follow-up. So feel free to resolve this comment

Reply:
@carmocca I'm not sure what this is used for. All I see is `self.resume_from_checkpoint_fit_path` being assigned a value but never being used. I think these users should access `.resume_checkpoint_path` instead